packages/chrony/chronyd.service

```
@@ -1,12 +1,12 @@
 [Unit]
 Description=A versatile implementation of the Network Time Protocol
 Documentation=https://chrony.tuxfamily.org
-After=network-online.target configured.target
-Requires=network-online.target configured.target
+After=network-online.target
```

This should still come after `preconfigured.target`:

```suggestion
After=network-online.target preconfigured.target
```